### PR TITLE
sc-69242-create-how-to-category-in-qml-repository

### PR DESCRIPTION
**Pull request by ashishks0522**
## Changes
- Add `How-To` category in JSON schema

### CODE REVIEW

**Comment by ashishks0522** [comment]
@ikurecic if you could help review and confirm how we want the `How-To` category copy to be?

**Review by doctorperceptron — APPROVED** [review]
LGTM. Will let @ikurecic sign-off on the formatting for "How-To".

File: README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ quantum computing paper/result.
   - Make sure the file name is `<name of your tutorial>.metadata.json`.
   - The "id" of the author will be the same as the one you chose when creating the bio. 
   - The date of publication and modification. Leave them empty in case you don't know them.
-  - Choose the categories your demo fits into: `"Getting Started"`, `"Optimization"`, `"Quantum Machine Learning"`, `"Quantum Chemistry"`, `"Devices and Performance"`, `"Quantum Computing"`, `"Quantum Hardware"` or `"Algorithms"`. Feel free to add more than one.
+  - Choose the categories your demo fits into: `"Getting Started"`, `"Optimization"`, `"Quantum Machine Learning"`, `"Quantum Chemistry"`, `"Devices and Performance"`, `"Quantum Computing"`, `"Quantum Hardware"`, `"Algorithms"` or `"How-To"`. Feel free to add more than one.
```
Comment (ikurecic) [inline README.md:107]:
```suggestion
  - Choose the categories your demo fits into: `"Getting Started"`, `"Optimization"`, `"Quantum Machine Learning"`, `"Quantum Chemistry"`, `"Devices and Performance"`, `"Quantum Computing"`, `"Quantum Hardware"`, `"Algorithms"` or `"How-to"`. Feel free to add more than one.
```

File: metadata_schemas/demo.metadata.schema.0.1.1.json [hunk 2]
```diff
@@ -51,7 +51,8 @@
             "Quantum Chemistry",
             "Devices and Performance",
             "Quantum Computing",
-            "Quantum Hardware"
+            "Quantum Hardware",
+            "How-To"
```
Comment (ikurecic) [inline metadata_schemas/demo.metadata.schema.0.1.1.json:55]:
```suggestion
            "How-to"
```

**Review by ikurecic — COMMENTED** [review]
Thank you for the tag.
What would be a good time to start adding this category to existing PennyLane Demos?

**Comment by ashishks0522** [comment]
> Thank you for the tag. What would be a good time to start adding this category to existing PennyLane Demos?

As soon as this PR is merged in, we should be good to start using this category

**Comment by ikurecic** [comment]
> > Thank you for the tag. What would be a good time to start adding this category to existing PennyLane Demos?
> 
> As soon as this PR is merged in, we should be good to start using this category

Thanks. As discussed at the pennylane.ai meeting today, I can have a pull request ready to assign this category to existing PennyLane Demos by 25 July.
Would it be best if I committed as part of this pull request, @ashishks0522 ?

**Comment by ashishks0522** [comment]
> Thanks. As discussed at the pennylane.ai meeting today, I can have a pull request ready to assign this category to existing PennyLane Demos by 25 July. Would it be best if I committed as part of this pull request, @ashishks0522 ?

I think we can just merge this one in to have the schema changes in `master` then you can create a pull request from `master` and continue with the work, since github checks might run based on what's in the `master` branch, that way it won't fail for this new category

**Review by ikurecic — APPROVED** [review]
Sure, @ashishks0522 , if that sounds good to you, let's do that. Thanks.

**Pull request merged**